### ISSUE TITLE
Cannot reproduce results

### ISSUE THREAD

**Issue by OlgaChaganova** [Fabian-Sc85/non-invasive-bp-estimation-using-deep-learning#19]
Hi! First of all, thank you for your work! The paper is well written and the code is very useful.

I have tried to reproduce your results in two ways:

1) Take the dataset you published on Zenodo and train your model (especially resnet) on that data. The metrics I got are the same as the ones you reported in the paper. Tensorboard tag on the image below: FABIAN_REPRODUSING

2) Run the whole algorithm, which includes downloading the raw MIMIC-III dataset, preparing it, and then training your ResNet model (steps 1-4 described in the README). In this case, the metrics are different from those given in the paper, and the loss behaviour on the validation dataset makes me think there is overfitting. Tensorboard tag on the image below: OUR DATA

Tensorboard graphics:

![image](https://github.com/Fabian-Sc85/non-invasive-bp-estimation-using-deep-learning/assets/68281687/e3cb96c3-ec1e-4141-a660-57a32efd4e7b)

I suspect that the reason for the discrepancy in the metrics lies in the data. Could you please say that the `prepare_MIMIC_dataset.py` is the same one you use for the dataset published on Zenodo? Another detail, you stated that you filtered PPG with low SNR in your paper, but this filtering is not done in `prepare_MIMIC_dataset.py`.

**Comment by jungyin** [comment]
May I ask if there has been any progress in the work? In the test, I extracted all RPPG signals with blood pressure below 120 from RPPG and found that many signals were fitted to high pressure above 150 or even 170, which left me quite confused
